### PR TITLE
Remove docs and update links to Tech Docs Template

### PR DESCRIPTION
**Pull request by m-green**
Now that we've moving developer docs to the Tech Docs Template (TDT), this 

- replaces moved docs with a redirect message (I'll raise a separate issue to remove these markdown files completely in 3 months or so)
- updates links across the docs so they point to the right place

I haven't changed links in the changelog.

### CODE REVIEW

File: docs/installation/installing-from-dist.md [hunk 1]
```diff
@@ -1,53 +1 @@
...
-  </head>
-  <body>
-    <!-- Copy and paste component HTML-->
-    <button class="govuk-button">This is a button component</button>
-    <script src="javascript/govuk-frontend-[latest version].min.js"></script>
-    <script>window.GOVUKFrontend.initAll()</script>
-  </body>
-</html>
-```
+This documentation has moved to [https://frontend.design-system.service.gov.uk/install-from-dist/](https://frontend.design-system.service.gov.uk/install-from-dist/).
```
Comment (hannalaakso) [inline docs/installation/installing-from-dist.md:1]: Should be `installing-from-dist`

File: README.md [hunk 2]
```diff
@@ -26,12 +26,12 @@ in your service.
...
 
 We recommend [installing GOV.UK Frontend using node package manager
-(npm)](docs/installation/installing-with-npm.md).
+(npm)](https://frontend.design-system.service.gov.uk/installing-with-npm/).
 
 ### 2. Install by using compiled files
 
 You can also [download the compiled and minified assets (CSS, JavaScript) from
-GitHub](docs/installation/installing-from-dist.md).
+GitHub](https://frontend.design-system.service.gov.uk/install-from-dist/).
```
[the remark this answers is not in the document]
Comment (m-green) [reply README.md]: Thanks for spotting all these @hannalaakso. All should be fixed in the latest commit.

File: README.md [hunk 3]
```diff
@@ -50,10 +50,10 @@ Using Frontend will help your service meet [level AA of WCAG 2.1](https://www.go
 
 You should also use:
 
-- [the JavaScript from GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
-- [a separate stylesheet](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md) if you support Internet Explorer 8
+- [the JavaScript from GOV.UK Frontend](ttps://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#javascript)
```
[the remark this answers is not in the document]
Comment (m-green) [reply README.md]: Fixed in https://github.com/alphagov/govuk-frontend/pull/1783/commits/96e909ccc0da33f1ae252f2c383215812035d2db

File: docs/contributing/versioning.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ The other primary way is through what is [published to npm](https://github.com/a
 
 This includes:
 
-- [JavaScript](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
+- [JavaScript](https://frontend.design-system.service.gov.uk/installing-with-npm/#javascript)
```
[the remark this answers is not in the document]
Comment (m-green) [reply docs/contributing/versioning.md]: Fixed in https://github.com/alphagov/govuk-frontend/pull/1783/commits/96e909ccc0da33f1ae252f2c383215812035d2db

File: docs/installation/installing-from-dist.md (hunk 1 above)
Comment (m-green) [reply docs/installation/installing-from-dist.md]: Fixed in https://github.com/alphagov/govuk-frontend/pull/1783/commits/96e909ccc0da33f1ae252f2c383215812035d2db

**Pull request merged**